nilearn/image/tests/test_image.py

```diff
@@ -852,12 +853,29 @@ def test_binarize_img(img_4d_rand_eye):
 
     assert_array_equal(np.unique(img2.dataobj), np.array([0, 1]))
     # Test that manual binarization equals binarize_img results.
-    img3 = img_4d_rand_eye
+    img3 = copy_img(img_4d_rand_eye)
     img3.dataobj[img_4d_rand_eye.dataobj < 0.5] = 0
     img3.dataobj[img_4d_rand_eye.dataobj >= 0.5] = 1
 
     assert_array_equal(img2.dataobj, img3.dataobj)
 
+    # Test option to use original or absolute values
+    img4_data = img_4d_rand_eye.get_fdata()
+    # Create a mask for half of the values and them negative
+    neg_mask = np.random.choice(
+        [True, False], size=img_4d_rand_eye.shape, p=[0.5, 0.5]
+    )
+
+    img4_data[neg_mask] *= -1
+    img4 = new_img_like(img_4d_rand_eye, img4_data)
+    # Binarize using original and absolute values
+    img4_original = binarize_img(img4, threshold=0, two_sided=False)
+    img4_absolute = binarize_img(img4, threshold=0, two_sided=True)
+    # Check that all values are 1 for absolute valued threshold
+    assert_array_equal(np.unique(img4_absolute.dataobj), np.array([1]))
```

Comment:
```suggestion
    assert_array_equal(np.unique(img4_absolute.get_fdata()), np.array([1]))
``` (just for the sake of homogeneity)

Reply:
Just made this change in a separate commit (had to fix a typo too)